packages/venia-concept/src/components/PurchaseDetailsPage/OrderItemsList/__tests__/orderItemsList.spec.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { List } from '@magento/peregrine';
+import OrderItemsList from '../OrderItemsList';
+import OrderItem from '../../OrderItem';
+
+configure({ adapter: new Adapter() });
+
+const classes = {
+    header: 'header'
+};
+
+const itemsMock = [
```

Comment:
Do we really need 3 items?! Maybe we can reduce number of items to 2, just to have less lines of code in this file.